stable/insights-agent/values.yaml

```diff
@@ -39,8 +39,8 @@ kubehunter:
   enabled: true
   schedule: "rand * * * *"
   image:
-    repository: quay.io/fairwinds/kube-hunter
-    tag: "1.0.0"
+    repository: aquasec/kube-hunter
+    tag: "501"
```

Comment:
Probably update `kubehunter.image.repository` and `kubehunter.image.tag` in `README` as well

Reply:
Good point!